packages/cerebral-website-tutorial/parts/01/public/index.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="blaze.min.css">
```

Comment:
i had that before, but i've wanted less dependencies. thats why i've downloaded it